common/djangoapps/mailchimp_pipeline/tasks.py

```diff
@@ -12,20 +12,21 @@
 
 
 @task()
-def update_org_details_at_mailchimp(org_label, org_type, work_area, list_id):
+def update_org_details_at_mailchimp(org_label, org_type, work_area, org_id,  list_id):
     log.info("Task to send organization details to MailChimp")
     log.info(org_label)
 
-    extended_profiles = UserExtendedProfile.objects.filter(organization__label__iexact=org_label.lower()).values("user__email")
+    extended_profiles = UserExtendedProfile.objects.filter(organization_id=org_id).values("user__email")
```

Comment:
This will cause inconsistency with the rest of the file/module, so I avoided doing that and still strongly advice against it, since that is not in the scope of the ticket, too.

Reply:
As this method `update_org_details_at_mailchimp` has already inconsistency(line no 31) with quotes, As you are here, It would be great if you make it consistent (with single quotes as we have decided to use single instead of double). Whenever other methods will have change, we can update them as well.